app/models/hbx_enrollment.rb

```diff
@@ -2976,6 +2976,13 @@ def aptc_tax_household_enrollments
     end
   end
 
+  def tax_household_group
+    thh_enrs = tax_household_enrollments
+    return if thh_enrs.blank?
+
+    thh_enrs.first&.tax_household&.tax_household_group
```

Comment:
can we use most recent record...`thh_enrs.last` instead of `thh_enrs.first`. will that make any difference